src/Templates/src/templates/maui-blazor/MauiApp.1.csproj

```diff
@@ -62,14 +62,24 @@
         <PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="MS_EXT_VERSION" />
     </ItemGroup>
 
-  <!-- Build Properties must be defined within these property groups to ensure successful publishing
-       to the Mac App Store. See: https://aka.ms/maui-publish-app-store#define-build-properties-in-your-project-file -->
-  <PropertyGroup Condition="$(TargetFramework.Contains('-maccatalyst')) and '$(Configuration)' == 'Debug'">
-    <CodesignEntitlements>Platforms/MacCatalyst/Entitlements.Debug.plist</CodesignEntitlements>
-  </PropertyGroup>
-
-  <PropertyGroup Condition="$(TargetFramework.Contains('-maccatalyst')) and '$(Configuration)' == 'Release'">
-    <CodesignEntitlements>Platforms/MacCatalyst/Entitlements.Release.plist</CodesignEntitlements>
-    <UseHardenedRuntime>true</UseHardenedRuntime>
-  </PropertyGroup>
+    <!-- Build Properties must be defined within these property groups to ensure successful publishing
+         to the Mac App Store. See: https://aka.ms/maui-publish-app-store#define-build-properties-in-your-project-file -->
+    <!-- See https://aka.ms/maui-publish-app-store#add-entitlements for more information about adding entitlements.-->
+
+    <PropertyGroup Condition="$(TargetFramework.Contains('-maccatalyst'))">
+        <!-- Enable this value to use browser developer tools while debugging.-->
+        <!-- See https://aka.ms/blazor-hybrid-developer-tools -->
+        <EnableDefaultMacCatalystDebugEntitlements Condition="'$(EnableDefaultMacCatalystDebugEntitlements)' == ''">True</EnableDefaultMacCatalystDebugEntitlements>
+        <EnableDefaultMacCatalystReleaseEntitlements Condition="'$(EnableDefaultMacCatalystReleaseEntitlements)' == ''">True</EnableDefaultMacCatalystReleaseEntitlements>
```

Comment:
Do these properties need to be in the `maccatalyst` workload instead of in customer projects?

You can use BlazorWebView in a non-MAUI app on MacCatalyst, example:

https://github.com/jonathanpeppers/spice/tree/main/samples/Spice.BlazorSample

Should it also be available in `dotnet new maccatalyst` project templates?